client/question/components/CodeSnippetDirective.js

```diff
@@ -23,19 +23,24 @@ tie.directive('codeSnippet', [function() {
       getContent: '&content'
     },
     template: `
-      <span ng-repeat="line in snippetLines">
+      <span ng-repeat="line in abbreviatedSnippetLines track by $index">
         <span class="tie-code-snippet-line">{{line}}</span>
         <br>
       </span>
+      <span ng-if="abbreviatedSnippetLines.length <= 3">
```

Comment:
Is there any reason this is a 3? If so, could we make it a constant?

Numbers by themselves worry me.

Reply:
I think this just follows @rabidbit's mocks. But yeah, good point, I'll constant-ize it.

Reply:
Are you referring to the user's dialogue (user code in conversation)? If so, we should only show three lines or so--just enough for the user to see a small snippet of code before clicking to open the modal/pop-up to view the whole code.

Reply:
Done. (And yep, I pulled 3 out into a constant along the lines of MAX_NUM_LINES_TO_SHOW_IN_ABBREVIATED_SNIPPET.)